pkgs/development/python-modules/pulumi/default.nix

```diff
@@ -22,9 +22,9 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     semver
-    protobuf
+    protobuf4
```

Comment:
Please don't propagate such a versioned package, since it causes a PYTHONPATH conflict.
Can we use `pythonRelaxDeps` instead? 

> Only unversioned attributes (e.g. pydantic, but not pypdantic_1) can be included in dependencies, since due to PYTHONPATH limitations we can only ever support a single version for libraries without running into duplicate module name conflicts.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Thanks for the review!
I see. AFAIK this package does not build with the latest version of Protobuf because of the compatibility with the abi, and all the dependent packages seem to be building now.
I will check if it is possible to change this, but I am unsure if it is fixable.

Reply:
I tried some options, but they do not seem to work.
If this PR does not meet the standards, I am happy to close this.
We can wait for upstream to support protobuf5 https://github.com/pulumi/pulumi/issues/16828.

Reply:
Thanks for investigating.
Unfortunately, we need to mark this package as broken for now.
```
meta = {
  ...
  # https://github.com/pulumi/pulumi/issues/16828
  broken = versionAtLeast protobuf.version "5";
};
```

Reply:
I got it. I will close this and re-open a PR.